### ISSUE TITLE
bv.get_section_by_name broken in most recent dev build

### ISSUE THREAD

**Issue by mattz0rt** [Vector35/binaryninja-api#1165]
This call was working before updating to the most recent dev build. I get the following error:
```
>>> bv.get_section_by_name('.text')
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "C:\Program Files\Vector35\BinaryNinja\plugins\..\python\binaryninja\binaryview.py", line 3698, in get_section_by_name
    result = Section(section)
  File "C:\Program Files\Vector35\BinaryNinja\plugins\..\python\binaryninja\binaryview.py", line 552, in __init__
    self.handle = core.handle_of_type(handle, core.BNSection)
  File "C:\Program Files\Vector35\BinaryNinja\plugins\..\python\binaryninja\_binaryninjacore.py", line 12733, in handle_of_type
    raise ValueError('expected pointer to %s' % str(handle_type))
ValueError: expected pointer to <class 'binaryninja._binaryninjacore.BNSection'>
```
From pdb it looks like "value" is an instance of "handle_type" instead of a pointer to it:
```
(Pdb) isinstance(value, ctypes.POINTER(handle_type))
False
(Pdb) isinstance(value, handle_type)
True
```

**Comment by mattz0rt** [comment]
I see that this is already being addressed, please ignore

**Comment by plafosse** [comment]
Its fine I'll reopen it until the issue is *actually* fixed. Making this fix actually exposed a refcount issue that I haven't yet sorted out.

**Comment by plafosse** [comment]
This was fixed in v1.1.1357
